gammapy/cube/tests/test_psf_map.py

```diff
@@ -22,13 +22,33 @@ def fake_psf3d(sigma=0.15 * u.deg):
     O, R, E = np.meshgrid(offsets, rad, energy)
 
     Rmid = 0.5 * (R[:-1] + R[1:])
-    gaus = np.exp(-0.5 * Rmid ** 2 / sigma ** 2)
+    if shape == "gauss":
+        val = np.exp(-0.5 * Rmid ** 2 / sigma ** 2)
+    else:
+        val = Rmid < sigma
     drad = 2 * np.pi * (np.cos(R[:-1]) - np.cos(R[1:])) * u.Unit("sr")
-    psf_values = gaus / ((gaus * drad).sum(0)[0])
+    psf_values = val / ((val * drad).sum(0)[0])
 
     return PSF3D(energy_lo, energy_hi, offsets, rad_lo, rad_hi, psf_values)
 
 
+def fake_aeff2d(area=1e6 * u.m ** 2):
+    offsets = np.array((0.0, 1.0, 2.0, 3.0)) * u.deg
```

Comment:
No need to manually call `np.array`, you can just put `[0.0, 1.0, 2.0, 3.0] * u.deg` and it'll make the same Quantity object.